Csg072020

csgrant00 opened the pull request:
added aip journals to config.py (and updated iop/oup ones)
updates to aip.py I think.

seasidesparrow commented:
@csgrant00 this is failing tests -- I made a change in master to remove a
dependency on adsputils, and it's failing.  Can you rebase from the current
master first?

On Mon, Aug 31, 2020 at 3:28 PM Carolyn Grant <notifications@github.com>
wrote:

> @csgrant00 <https://github.com/csgrant00> requested your review on: #98
> <https://github.com/adsabs/adsabs-pyingest/pull/98> Csg072020.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/adsabs/adsabs-pyingest/pull/98#event-3711313385>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD7KTZQG7AZ35VDNFEBWLRLSDP2W3ANCNFSM4QQ3OHOA>
> .
>


-- 
Matthew Templeton
Astrophysics Data System
Center for Astrophysics | Harvard & Smithsonian
MS83, 60 Garden Street
Cambridge, MA 02138
ORCID: 0000-0003-1918-0622
matthew.templeton@cfa.harvard.edu <matthew.templeton@mail.cfa.harvard.edu>

csgrant00 commented:
OK, can you explain this?

jasra-130: git remote add upstream
https://github.com/adsabs/adsabs-pyingest.git

jasra-131: git pull --rebase upstream master

error: Cannot pull with rebase: You have unstaged changes.

jasra-132: git status

On branch csg072020

Changes not staged for commit:

  (use "git add <file>..." to update what will be committed)

  (use "git checkout -- <file>..." to discard changes in working directory)


modified:   pyingest/parsers/oup.py


no changes added to commit (use "git add" and/or "git commit -a")

jasra-133: git add pyingest/parsers/oup.py

jasra-135: git commit -a

[csg072020 60e1392] fixed type in pdf path

 1 file changed, 1 insertion(+), 1 deletion(-)

jasra-136: git pull --rebase upstream master

From https://github.com/adsabs/adsabs-pyingest

 * branch            master     -> FETCH_HEAD

 * [new branch]      master     -> upstream/master

First, rewinding head to replay your work on top of it...

Applying:  new file:   pyingest/parsers/crossref.py

Applying:  modified:   examples/ex_proquest.py

Applying:  modified:   requirements.txt

Applying:  modified:   pyingest/config/utils.py

Applying:  modified:   pyingest/parsers/aip.py

Applying: added new iop journals

Applying: added parsing example scripts

Applying: added aip parsing (work in progress)

Applying: fixed type in pdf path

jasra-137: git push origin master

Password for 'https://csgrant00@github.com':

Everything up-to-date

jasra-138:



Q1: Why when I look at the repo in my browser am I 30 commits behind master?


Q2: It looks like I'm good to go - do I do a new pull request since the
last one

failed?  Or do I somehow re-pull that one? (I did add the one other commit)


Thanks!

-Carolyn


*************************************************************************

 Carolyn Stern Grant              Astrophysics Data System (ADS)

 cgrant@cfa.harvard.edu        Center for Astrophysics

 617-495-7154 (voicemail)      60 Garden Street  MS 83

 617-495-7356 fax                 Cambridge, MA  02138

*************************************************************************


On Tue, Sep 1, 2020 at 9:26 AM Matthew Templeton <notifications@github.com>
wrote:

> @csgrant00 this is failing tests -- I made a change in master to remove a
> dependency on adsputils, and it's failing. Can you rebase from the current
> master first?
>
> On Mon, Aug 31, 2020 at 3:28 PM Carolyn Grant <notifications@github.com>
> wrote:
>
> > @csgrant00 <https://github.com/csgrant00> requested your review on: #98
> > <https://github.com/adsabs/adsabs-pyingest/pull/98> Csg072020.
> >
> > —
> > You are receiving this because your review was requested.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/adsabs/adsabs-pyingest/pull/98#event-3711313385>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AD7KTZQG7AZ35VDNFEBWLRLSDP2W3ANCNFSM4QQ3OHOA
> >
> > .
> >
>
>
> --
> Matthew Templeton
> Astrophysics Data System
> Center for Astrophysics | Harvard & Smithsonian
> MS83, 60 Garden Street
> Cambridge, MA 02138
> ORCID: 0000-0003-1918-0622
> matthew.templeton@cfa.harvard.edu <matthew.templeton@mail.cfa.harvard.edu>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/adsabs/adsabs-pyingest/pull/98#issuecomment-684851692>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABKDRFOLFKCGA5N5IZXKZF3SDTY77ANCNFSM4QQ3OHOA>
> .
>

seasidesparrow commented:
I think you have two different repos in different states.  You made the
pull request for a branch of adsabs/adsabs-pyingest called Csg072020 --
which is a branch off of the upstream repository.  Master on that repo is
what is our primary repository.  But you've also got a personal fork (which
is what you should be using) in csgrant00/adsabs-pyingest.  You need to
make edits and rebase that one, and make a PR from
csgrant00/adsabs-pyingest:master rather than
adsabs/adsabs-pyingest:Csg072020.

On Tue, Sep 1, 2020 at 9:55 AM Carolyn Grant <notifications@github.com>
wrote:

> OK, can you explain this?
>
> jasra-130: git remote add upstream
> https://github.com/adsabs/adsabs-pyingest.git
>
> jasra-131: git pull --rebase upstream master
>
> error: Cannot pull with rebase: You have unstaged changes.
>
> jasra-132: git status
>
> On branch csg072020
>
> Changes not staged for commit:
>
> (use "git add <file>..." to update what will be committed)
>
> (use "git checkout -- <file>..." to discard changes in working directory)
>
>
> modified: pyingest/parsers/oup.py
>
>
> no changes added to commit (use "git add" and/or "git commit -a")
>
> jasra-133: git add pyingest/parsers/oup.py
>
> jasra-135: git commit -a
>
> [csg072020 60e1392] fixed type in pdf path
>
> 1 file changed, 1 insertion(+), 1 deletion(-)
>
> jasra-136: git pull --rebase upstream master
>
> From https://github.com/adsabs/adsabs-pyingest
>
> * branch master -> FETCH_HEAD
>
> * [new branch] master -> upstream/master
>
> First, rewinding head to replay your work on top of it...
>
> Applying: new file: pyingest/parsers/crossref.py
>
> Applying: modified: examples/ex_proquest.py
>
> Applying: modified: requirements.txt
>
> Applying: modified: pyingest/config/utils.py
>
> Applying: modified: pyingest/parsers/aip.py
>
> Applying: added new iop journals
>
> Applying: added parsing example scripts
>
> Applying: added aip parsing (work in progress)
>
> Applying: fixed type in pdf path
>
> jasra-137: git push origin master
>
> Password for 'https://csgrant00@github.com':
>
> Everything up-to-date
>
> jasra-138:
>
>
>
> Q1: Why when I look at the repo in my browser am I 30 commits behind
> master?
>
>
> Q2: It looks like I'm good to go - do I do a new pull request since the
> last one
>
> failed? Or do I somehow re-pull that one? (I did add the one other commit)
>
>
> Thanks!
>
> -Carolyn
>
>
> *************************************************************************
>
> Carolyn Stern Grant Astrophysics Data System (ADS)
>
> cgrant@cfa.harvard.edu Center for Astrophysics
>
> 617-495-7154 (voicemail) 60 Garden Street MS 83
>
> 617-495-7356 fax Cambridge, MA 02138
>
> *************************************************************************
>
>
> On Tue, Sep 1, 2020 at 9:26 AM Matthew Templeton <notifications@github.com
> >
> wrote:
>
> > @csgrant00 this is failing tests -- I made a change in master to remove a
> > dependency on adsputils, and it's failing. Can you rebase from the
> current
> > master first?
> >
> > On Mon, Aug 31, 2020 at 3:28 PM Carolyn Grant <notifications@github.com>
> > wrote:
> >
> > > @csgrant00 <https://github.com/csgrant00> requested your review on:
> #98
> > > <https://github.com/adsabs/adsabs-pyingest/pull/98> Csg072020.
> > >
> > > —
> > > You are receiving this because your review was requested.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/adsabs/adsabs-pyingest/pull/98#event-3711313385>,
> or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AD7KTZQG7AZ35VDNFEBWLRLSDP2W3ANCNFSM4QQ3OHOA
> > >
> > > .
> > >
> >
> >
> > --
> > Matthew Templeton
> > Astrophysics Data System
> > Center for Astrophysics | Harvard & Smithsonian
> > MS83, 60 Garden Street
> > Cambridge, MA 02138
> > ORCID: 0000-0003-1918-0622
> > matthew.templeton@cfa.harvard.edu <
> matthew.templeton@mail.cfa.harvard.edu>
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/adsabs/adsabs-pyingest/pull/98#issuecomment-684851692>,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ABKDRFOLFKCGA5N5IZXKZF3SDTY77ANCNFSM4QQ3OHOA
> >
> > .
> >
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/adsabs/adsabs-pyingest/pull/98#issuecomment-684872037>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD7KTZX3BWOFBCVBXQNDDG3SDT4OHANCNFSM4QQ3OHOA>
> .
>


-- 
Matthew Templeton
Astrophysics Data System
Center for Astrophysics | Harvard & Smithsonian
MS83, 60 Garden Street
Cambridge, MA 02138
ORCID: 0000-0003-1918-0622
matthew.templeton@cfa.harvard.edu <matthew.templeton@mail.cfa.harvard.edu>
